conda_build.recipe/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: numpy
+  version: 1.11.0{{ '+' + environ.get('GIT_DESCRIBE_NUMBER', '0') + '.' + environ.get('GIT_DESCRIBE_HASH', '0') if environ.get('GIT_DESCRIBE_NUMBER', '0') != '0' else "" }}
```

Comment:
I believe I understood the problem, but I still don't believe it to be one. Devs that use NumPy have two choices:

1. Build your packages with the new version of NumPy.
2. Force install the newer version of NumPy.

If they are making use of new NumPy C API features, they will need to follow 1 anyways. If they are using Python only or are using older C API features, the can get away with doing 2. I am happy to provide example cases if that will help. :)